doc/tutorials/sr_basics.rst

```diff
@@ -0,0 +1,50 @@
+====================
+Structured Reporting
+====================
+
+.. versionadded:: 1.4
+
+Starting in *pydicom* version 1.4, some support for DICOM Structured Reporting (SR) began to be added,
+as alpha code; the API for this is subject to change in future *pydicom* versions. At this point the 
+code is limited to code dictionaries and one class :class:`~pydicom.sr.coding.Code` 
```

Comment:
`Code` needs to be added to the [API reference](https://github.com/pydicom/pydicom/blob/master/doc/reference/index.rst) for this to auto-link, maybe under a new `sr.rst` page?

Reply:
Yeah, I was trying to avoid that as there is so little to say about the class (basically a named tuple with equality checking), and it is alpha only.  This guide was necessary so people would have some idea what was available so far.  I'll check into creating a page anyway, see how it looks, make sure it is labeled alpha.  Otherwise I'll not try to link here.